### ISSUE TITLE
Mobile-Id REST API

### ISSUE THREAD

**Issue by rang501** [BitWeb/id-services#7]
SK is starting to shut down the current API. Will this library implement support for the new REST API near the future?

**Comment by tobre6** [comment]
Yes, but not in the near future unless @rang501 doesn't want to create PR?
